### ISSUE TITLE
Python3 not found in cmake configure process even though it is installed

### ISSUE THREAD

**Issue by DarthMalloc** [mgbellemare/Arcade-Learning-Environment#479]
A little while back, I posted an issue that I had while trying to install ALE for python3.8 with SDL support [https://github.com/mgbellemare/Arcade-Learning-Environment/issues/438](url). I am trying to do the same installation on a different computer, this time with Linux Mint 20.3. When I try to install, I get an error saying that the development files for Python3 are not found even though they are installed. I copied the exact terminal output into an update I posted to the old issue that is linked in this post. I greatly appreciate your help.

**Comment by JesseFarebro** [comment]
Hi @DarthMalloc,

Is there a reason you're compiling yourself instead of installing the pre-built wheel?

Regardless, I'll give some advice on how to debug this:

- CMake might not be finding the right Python installation. You can manually override this with `-DPython3_EXECUTABLE=$MY_PYTHON_EXECUTABLE_PATH`. This oftentimes solves this issue.
- If the above doesn't work can you post the output of `python -m sysconfig`.

**Comment by DarthMalloc** [comment]
Thanks! Does installing the pre-built wheel also provide SDL support? If so, I will just use that approach instead.

**Comment by JesseFarebro** [comment]
Yes! We package SDL with the prebuilt wheels. It should work out of the box.
